user_docs/en/userGuide.md

```diff
@@ -3223,7 +3223,7 @@ If speech rules for the main language can not be found, English ("en") is used.
 
 Different speech styles are used to change how math is reported.
 
-* ClearSpeak was developed by ETS for use on high-stakes tests such as the SAT.
+* ClearSpeak was developed by ETS (Educational Testing Service) for use on high-stakes tests such as the SAT.
```

Comment:
I think this order is more typical for explaining acronyms. Also lets try and remove US centric terms 😄 

```suggestion
* ClearSpeak was developed by the Educational Testing Service (ETS) for use on high-stakes tests such as national standardized testing.
```